### ISSUE TITLE
Experiments on cifar

### ISSUE THREAD

**Issue by MrChenFeng** [UMBCvision/MSF#8]
Thanks for your great work, quite interesting.
I wonder have you ever tried to experiment with small-scale datasets? Imagenet it rather big to try as you know.

Best.

**Comment by ajtejankar** [comment]
Hi Chen,

Unfortunately, no. We believe CIFAR is too small to be helpful for comparing SSL methods. You could use the ImageNet-100 subset (~125k) from [Contrastive Multiview Coding](https://github.com/HobbitLong/CMC/blob/master/imagenet100.txt). I have ran a modified version of MSF on this dataset for 500 epochs and obtained around 83% linear layer accuracy. Still, 500 epochs on ImageNet-100 is large compared to CIFAR, but it is necessary. Comparison of methods with less number of epochs ignores delayed convergence of some methods. There is no quick and easy path to get results in SSL research these days.

Thanks,
Ajinkya

**Comment by MrChenFeng** [comment]
Yes, exactly, I agree with your opinions for SSL... Thanks so much!
